transfer.md

```diff
@@ -1,16 +1,14 @@
-# PODMAN Usage Transfer
+# podman Usage Transfer
 
-This document outlines useful information for ops and dev transfer as it relates to infrastructure that utilizes PODMAN.
+This document outlines useful information for ops and dev transfer as it relates to infrastructure that utilizes `podman`.
 
 ## Operational Transfer
 
 ## Abstract
 
-The `crio` daemon is intended to provide the [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md) socket needed for Kubernetes to use for automating deployment, scaling, and management of containerized applications (See the document for [configuring kubernetes to use PODMAN](./kubernetes.md) for more information on that).
-Therefore the `crioctl` command line is a client that interfaces to the same grpc socket as the kubernetes daemon would, for talking to the `crio` daemon.
-In many ways `crioctl` is only as feature rich as the Kubernetes CRI requires.
-There are additional tools e.g. `podman` and [`buildah`](https://github.com/projectatomic/buildah) that provide a feature rich set of commands for all operational needs in a Kubernetes environment.
-
+podman is a tool for managing Pods, Containers, and Container Images.  The CLI
+for podman is based on the Docker CLI, although podman does not require a
+container daemon to be running in order to function.
```

Comment:
Maybe runtime daemon instead of container daemon?

Reply:
I'd pref "Podman" through out with the exception of commands.

Reply:
Fixed